mpc-aio/src/protocol/garble/exec/dual.rs

```diff
@@ -102,7 +91,7 @@ where
         let gc_ev = GarbledCircuit::<Partial>::from_msg(circ, msg)?;
         let labels_ev = self.label_receiver.receive_labels(inputs.to_vec()).await?;
 
-        let evaluated_gc = self.evaluator.evaluate(gc_ev, &labels_ev).await?;
+        let evaluated_gc = self.garbler.evaluate(gc_ev, labels_ev).await?;
```

Comment:
can this be called `dualex_party` instead of `garbler`? because `garbler.evaluate` doesn't sound right
Or better still, can we have `self ` methods `generate` and `evaluate` which internally call garbler/evaluator's generate/evaluate?